package.json

```diff
@@ -65,6 +65,8 @@
     "polka": "0.5.2",
     "preact": "10.3.4",
     "prop-types": "15.7.2",
+    "react-linkifier": "^4.0.0",
+    "react-linkify": "^1.0.0-alpha",
```

Comment:
Do we need both libs? Also, I prefer to pin exact versions for dependencies, just so it's the same across computers.

`npm i --save-exact react-linkify` etc.

Reply:
✔️